src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -81,20 +83,27 @@ To see the usage, use the `--help` parameter.
 ----
 $ ${HBASE_HOME}/bin/hbase canary -help
 
-Usage: bin/hbase org.apache.hadoop.hbase.tool.Canary [opts] [table1 [table2]...] | [regionserver1 [regionserver2]..]
+Usage: hbase org.apache.hadoop.hbase.tool.Canary [opts] [table1 [table2]...] | [regionserver1 [regionserver2]..]
  where [opts] are:
    -help          Show this help and exit.
    -regionserver  replace the table argument to regionserver,
       which means to enable regionserver mode
+   -allRegions    Tries all regions on a regionserver,
+      only works in regionserver mode.
+   -zookeeper    Tries to grab zookeeper.znode.parent
+      on each zookeeper instance
    -daemon        Continuous check at defined intervals.
```

Comment:
This option is missing from the help message:  

 ```
-permittedZookeeperFailures <N>    Ignore first N failures when attempting to 
      connect to individual zookeeper nodes in the ensemble```

Reply:
this was added to the master ref guide in HBASE-21320. let me see if all those related changes work for branch-1.

Reply:
The rest of the help output looked correct for branch-1 canary, only `-permittedZookeeperFailures` was missing. Master branch has a slightly different help message but the master reference guide is up to date with it.